numpy/core/tests/test_defchararray.py

```diff
@@ -345,6 +345,11 @@ def test_center(self):
         assert_(issubclass(C.dtype.type, np.string_))
         assert_array_equal(C, tgt)
 
+        C = np.char.center([b'a', b'b'], 3, fillchar=[b'a', b'b'])
```

Comment:
all of the new test cases is identical, How about extract to a new test case that named test_fillchar?

Reply:
Done as of https://github.com/numpy/numpy/pull/17084/commits/9114becee335138c6f888c244594c53b2f2d6830, the tests should now be a bit more varied.